testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1903,6 +1904,36 @@ public void testDeleteAfterMajorCompaction()
         });
     }
 
+    @Test
+    public void testUnbucketedTransactionalTableWithTaskWriterCountGreaterThan1()
+    {
+        withTemporaryTable("test_delete_with_unbucketed_transactional_table", true, false, NONE, tableName -> {
+            onTrino().executeQuery(format("CREATE TABLE %s  WITH (format='ORC', transactional=true) AS TABLE tpch.sf1000.orders WITH NO DATA", tableName));
+            onTrino().executeQuery("SET SESSION scale_writers = true");
+            onTrino().executeQuery("SET SESSION writer_min_size = '4kB'");
+            onTrino().executeQuery("SET SESSION task_writer_count = 4");
```

Comment:
Should we have a test with `task_writer_count>1` for unpartitioned (here) and also partitioned tables?
i think we should.